SRC/PORT.CPP

```diff
@@ -233,15 +262,28 @@ void tk_port_event_tick( void )
                 }
             }
                 break;
+
+            case SDL_WINDOWEVENT:
+                if (e.window.event == SDL_WINDOWEVENT_RESIZED) {
```

Comment:
Totally nitpicking, but opening bracket could be in next line like elsewhere.

Reply:
Fixed :)